Mathlib/RingTheory/Idempotents.lean

```diff
@@ -335,6 +345,143 @@ theorem CompleteOrthogonalIdempotents.of_ker_isNilpotent_of_isMulCentral
     simpa [RingHom.mem_ker, sub_eq_zero] using congr_fun h₂.symm i
   exact h₁
 
+omit [Fintype I] in
+lemma ne_sum_ortho {e : R} (he : IsPrimitiveIdempotents e)
```

Comment:
I don't understand what this lemma is for. Can you point me to an application?

Reply:
It's used in the later theorem I'm gonna push in the next PR about this

Reply:
hmmm actually u're right, deleted!